<ui version="4.0" >
 <class>GenericDBViewer</class>
 <widget class="QWidget" name="GenericDBViewer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="KListWidget" name="fDBList" />
     </item>
     <item>
      <widget class="KComboBox" name="fDBType" >
       <item>
        <property name="text" >
         <string>All Databases</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only Applications (*.prc)</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Only Databases (*.pdb)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Database Information</string>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" colspan="2" >
         <widget class="KTextEdit" name="fDBInfo" >
          <property name="autoFormatting" >
           <set>QTextEdit::AutoAll</set>
          </property>
          <property name="lineWrapMode" >
           <enum>QTextEdit::WidgetWidth</enum>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="KPushButton" name="fDBInfoButton" >
          <property name="text" >
           <string>&amp;General</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="KPushButton" name="fAppInfoButton" >
          <property name="text" >
           <string>App&amp;Info</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="fRecordList" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled" >
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled" >
        <bool>false</bool>
       </property>
       <property name="rowCount" >
        <number>0</number>
       </property>
       <property name="columnCount" >
        <number>3</number>
       </property>
       <column>
        <property name="text" >
         <string>Rec. Nr.</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text" >
         <string>Record ID</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" />
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
